ewkcoffea/data/run3_lep_sf/electron_sf/2022EE_ele/electron.json

```diff
@@ -729,6 +729,178 @@
                                                     ],
                                                     "flow": "error"
                                                 }
+                                            },
+                                            {
+                                                "key": "wp90noiso",
+                                                "value": {
+                                                    "nodetype": "multibinning",
+                                                    "inputs": [
+                                                        "eta",
+                                                        "pt"
+                                                    ],
+                                                    "edges": [
+                                                        [
+                                                            -Infinity,
```

Comment:
We never grab this variation... right?

Reply:
Yes, but this wasn't actually edited by me. I just grabbed the updated file.